ansible/roles/postgresql_server/tasks/manage_clusters.yml

```diff
@@ -65,6 +65,19 @@
   loop_control:
     label: '{{ item.item }}'
 
+- name: Log directory path
+  file:
+    name: "{{item.log_directory}}"
+    mode: 0750
```

Comment:
The octal value should be quoted to avoid issues with Jinja interpreting it and converting to decimal value.

Reply:
done.